src/components/board/index.js

```diff
@@ -41,15 +41,14 @@ Board.propTypes = {
 
 const styles = StyleSheet.create({
   body: {
-    margin: "0px",
-    padding: "100",
+    margin: 5,
+    padding: 10,
     display: "flex",
     flexWrap: "wrap",
     justifyContent: "center",
     flexDirection: "row",
-    width: "25%",
     alignItems: "center",
     alignContent: "space-between",
-    width: "450px",
+    width: 500,
```

Comment:
works at 600 on my end to render everything in rows without extras on any level, but may need to be adjusted for the phone app